test/y2storage/partition_id_test.rb

```diff
@@ -123,4 +123,68 @@
       end
     end
   end
+
+  describe "#sort_order" do
+    it "sorts Linux Native before Linux Swap" do
+      linux = Y2Storage::PartitionId::LINUX
+      swap = Y2Storage::PartitionId::SWAP
+      expect(linux.sort_order < swap.sort_order).to eq true
```

Comment:
NP: You also can use 

```ruby
expect(linux.sort_order).to be < swap.sort_order
```

which seems more readable to me.

Reply:
OK, good idea.